infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -10,12 +10,26 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/{{ inventory_hostname }}/privkey.pem; # managed by Certbot
 
     ssl on;
-    ssl_session_cache  builtin:1000  shared:SSL:10m;
     ssl_protocols  TLSv1.2 TLSv1.3;
     ssl_ciphers EECDH+AESGCM:EDH+AESGCM;
     ssl_prefer_server_ciphers on;
     access_log            /var/log/nginx/java_server.access.log;
 
+    ssl_prefer_server_ciphers on; 
+    ssl_dhparam /etc/nginx/dhparam.pem; # openssl dhparam -out /etc/nginx/dhparam.pem 4096
```

Comment:
Just wanted to point out that this file doesn't need to be a secret according to this so question:
https://security.stackexchange.com/questions/94390/whats-the-purpose-of-dh-parameters

It's just some precomputed values.

Reply:
(Because sou called it "dhparam private key")